### PR TITLE
feat: fix alignment

### LINKED ISSUE #180: fix alignment
align this:
image

### PR DESCRIPTION
**Pull request by zugdev**
Resolves #180

image
image

### CODE REVIEW

File: static/style/inverted-style.css [hunk 1]
```diff
@@ -360,18 +360,22 @@
...
     -webkit-backdrop-filter: blur(8px);
     align-items: center;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
   }
   #authentication {
     position: absolute;
-    right: 0;
+    right: 5px;
```
Comment (0x4007) [inline static/style/inverted-style.css]: Doesn't seem right should be a multiple of 4px

File: static/style/style.css [hunk 2]
```diff
@@ -360,18 +360,22 @@
   }
   .toolbar {
     position: fixed;
-    width: 100vw;
+    width: 100%;
     height: 48px;
     top: 0;
+    padding: 0 5px;
```
Comment (0x4007) [inline static/style/style.css]: Same

File: static/style/style.css [hunk 3]
```diff
@@ -360,18 +360,22 @@
...
     -webkit-backdrop-filter: blur(8px);
     align-items: center;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
   }
   #authentication {
     position: absolute;
-    right: 0;
+    right: 5px;
```
Comment (0x4007) [inline static/style/style.css]: Same

**Comment by 0x4007** [comment]
Can you try to match the spacing of the left side more closely

File: static/style/inverted-style.css (hunk 1 above)
Comment (zugdev) [reply static/style/inverted-style.css]: there is a 1px border in the issue display

**Comment by 0x4007** [comment]
The fastest way to handle this is if I just do it but basically the right side spacing does not match the left side (devpool directory header) spacing.

**Comment by zugdev** [comment]
I'll do it

**Comment by zugdev** [comment]
should be good now @0x4007

**Comment by zugdev** [comment]
when I get some time later I will remove absolute from authenticated and the padding will work for both view-toggle and authenticated. I also plan in handling the username in a better way

**Comment by zugdev** [comment]
image

**Review by 0x4007 — APPROVED** [review]
Looks better when I tested from mobile. Will test on desktop another day.

**Comment by 0x4007** [comment]
> when I get some time later I will remove absolute from authenticated 

I did this as a fix because when it would load it would move elements on the header bar but if we can set a fixed width (the usernames are variable length) then it should render fine.

**Pull request merged**